sql/hive/src/test/resources/golden/constant object inspector for generic udf-0-cc120a2331158f570a073599985d3f55

```diff
@@ -1 +1 @@
-{"aa":"10","aaaaaa":"11","aaaaaa":"12","bb12":"13","s14s14":"14"}
+{"aa":"10","aaaaaa":"11","aaaaaa":"12","Bb12":"13","s14s14":"14"}
```

Comment:
The query is:
```scala
createQueryTest("constant object inspector for generic udf",
    """SELECT named_struct(
      lower("AA"), "10",
      repeat(lower("AA"), 3), "11",
      lower(repeat("AA", 3)), "12",
      printf("Bb%d", 12), "13",
      repeat(printf("s%d", 14), 2), "14") FROM src LIMIT 1""")
```
Since `printf` in Hive didn't change word case in `Bb%d`, therefore, `Bb12` is the right answer